### PR TITLE
new beatmap layout

### PR DESCRIPTION
**Pull request by dhcdht**

### CODE REVIEW

**Comment by isenmann** [comment]
I see your intention for this pull request. I have left out the _beatmapCharacteristicName intentionally because I didn't saw any need for it at the moment. Is this used somehow in the original BeatSaber or the mods? All song files I have looked at have only one _beatmapCharacteristicName all the time.

I will try the button stuff this evening but it looks good, something more generic than my hardcoded stuff for only four difficulties.

Is it possible that you do the pull request to the develop branch again? Changes on the master branch will only be done if a new release is out (I use the Gitflow Workflow model).

File: Assets/_Scripts/MainMenu.cs [hunk 1]
```diff
@@ -162,45 +162,37 @@ public void LoadSong()
...
 
                 button.GetComponentInChildren<Text>().text = difficulty;
-                button.GetComponentInChildren<Button>().onClick.AddListener(() => StartSceneWithDifficulty(difficulty));
+                button.GetComponentInChildren<Button>().onClick.AddListener(() => StartSceneWithDifficulty(0, difficulty));
                 button.SetActive(true);
                 buttonsCreated.Add(button);
             }
 
-            switch (buttonsCreated.Count)
+            float leftAlign = (-250-36) * (buttonsCreated.Count/2);
```
Comment (isenmann) [inline Assets/_Scripts/MainMenu.cs]: Thanks for that, the button stuff works like expected!!

**Comment by dhcdht** [comment]
`Is this used somehow in the original BeatSaber or the mods?`
Yes , It is used in the original BeatSaber. I have a pic for you
image

`All song files I have looked at have only one _beatmapCharacteristicName all the time`
It must be other version. here is my song file
[Info.txt](https://github.com/isenmann/OpenSaberVR/files/3412991/Info.txt)
You will find `"_version":"2.0.0"` and `"_difficultyBeatmapSets":[{"_beatmapCharacteristicName":"Standard","_difficultyBeatmaps":[{"_difficulty":"Easy"`
I use Best Saber 1.1.0

**Comment by isenmann** [comment]
Now I got it. Okay, all the files I had we're from the mod websites and there the most has only one _beatmapCharacteristicName. 

It's used for the different types of the sabers, one or two handed and so on. I will place the support for one saber and the other game type to the enhancement list, so I won't forget it.

I will integrate your pull request manually into the develop branch. Or if you want you can change or create a new pull request against the develop branch. I won't merge it into the master branch for the mentioned reasons in my last comment.

Thanks for your changes. 👍

**Comment by dhcdht** [comment]
I already change my pull request to develop branch yesterday.
Sorry I didn’t mention it in my last comment.

**Pull request merged**